### ISSUE TITLE
[foxy][tutorial] tb3_simulation_launch.py not launching properly

### ISSUE THREAD

**Issue by mag-seasony** [ros-planning/navigation2#2094]
## Bug report

**Required Info:**

- Operating System: Ubuntu 20.04
- ROS2 Version: Foxy binaries
- Version or commit hash: ros-foxy-navigation2	0.4.3-1focal.20201103.171126
- DDS implementation: not sure

#### Steps to reproduce issue
As specified in [this](https://navigation.ros.org/getting_started/index.html) tutorial:

```
source /opt/ros/<ros2-distro>/setup.bash
export TURTLEBOT3_MODEL=waffle
export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:/opt/ros/<ros2-distro>/share/turtlebot3_gazebo/models
```
then
```
ros2 launch nav2_bringup tb3_simulation_launch.py
```

#### Expected behavior
turtlebot 3 in a simulated environment, with the navigation stack up and running

#### Actual behavior
Gazebo fail to launch properly (get stuck then do not load the sdf), and the navigation stack does not run with the following error:
```
Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
```

#### Additional information

all the output from the launch:
```
os2 launch nav2_bringup tb3_simulation_launch.py
[INFO] [launch]: All log files can be found below /home/mg/.ros/log/2020-11-16-14-53-06-981842-varus-60948
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [gzserver-1]: process started with pid [60950]
[INFO] [gzclient-2]: process started with pid [60952]
[INFO] [robot_state_publisher-3]: process started with pid [60954]
[INFO] [rviz2-4]: process started with pid [60956]
[INFO] [map_server-5]: process started with pid [60958]
[INFO] [amcl-6]: process started with pid [60960]
[INFO] [lifecycle_manager-7]: process started with pid [60962]
[INFO] [controller_server-8]: process started with pid [60965]
[INFO] [planner_server-9]: process started with pid [60992]
[INFO] [recoveries_server-10]: process started with pid [61006]
[INFO] [bt_navigator-11]: process started with pid [61016]
[INFO] [waypoint_follower-12]: process started with pid [61022]
[INFO] [lifecycle_manager-13]: process started with pid [61027]
[map_server-5] [INFO] [map_io]: Loading yaml file: /opt/ros/foxy/share/nav2_bringup/maps/turtlebot3_world.yaml
[map_server-5] [DEBUG] [map_io]: resolution: 0.05
[map_server-5] [DEBUG] [map_io]: origin[0]: -10
[map_server-5] [DEBUG] [map_io]: origin[1]: -10
[map_server-5] [DEBUG] [map_io]: origin[2]: 0
[map_server-5] [DEBUG] [map_io]: free_thresh: 0.196
[map_server-5] [DEBUG] [map_io]: occupied_thresh: 0.65
[map_server-5] [DEBUG] [map_io]: mode: trinary
[map_server-5] [DEBUG] [map_io]: negate: 0
[map_server-5] [INFO] [map_io]: Loading image_file: /opt/ros/foxy/share/nav2_bringup/maps/turtlebot3_world.pgm
[map_server-5] [DEBUG] [map_io]: Read map /opt/ros/foxy/share/nav2_bringup/maps/turtlebot3_world.pgm: 384 X 384 map @ 0.05 m/cell
[robot_state_publisher-3] [WARN] [1605534787.283666952] [robot_state_publisher]: No robot_description parameter, but command-line argument available.  Assuming argument is name of URDF file.  This backwards compatibility fallback will be removed in the future.
[robot_state_publisher-3] Parsing robot urdf xml string.
[robot_state_publisher-3] Link base_link had 7 children
[robot_state_publisher-3] Link camera_link had 2 children
[robot_state_publisher-3] Link camera_depth_frame had 1 children
[robot_state_publisher-3] Link camera_depth_optical_frame had 0 children
[robot_state_publisher-3] Link camera_rgb_frame had 1 children
[robot_state_publisher-3] Link camera_rgb_optical_frame had 0 children
[robot_state_publisher-3] Link caster_back_left_link had 0 children
[robot_state_publisher-3] Link caster_back_right_link had 0 children
[robot_state_publisher-3] Link imu_link had 0 children
[robot_state_publisher-3] Link base_scan had 0 children
[robot_state_publisher-3] Link wheel_left_link had 0 children
[robot_state_publisher-3] Link wheel_right_link had 0 children
[robot_state_publisher-3] [INFO] [1605534787.285548328] [robot_state_publisher]: got segment base_footprint
[robot_state_publisher-3] [INFO] [1605534787.285578734] [robot_state_publisher]: got segment base_link
[robot_state_publisher-3] [INFO] [1605534787.285586840] [robot_state_publisher]: got segment base_scan
[robot_state_publisher-3] [INFO] [1605534787.285593440] [robot_state_publisher]: got segment camera_depth_frame
[robot_state_publisher-3] [INFO] [1605534787.285601384] [robot_state_publisher]: got segment camera_depth_optical_frame
[robot_state_publisher-3] [INFO] [1605534787.285606536] [robot_state_publisher]: got segment camera_link
[robot_state_publisher-3] [INFO] [1605534787.285611498] [robot_state_publisher]: got segment camera_rgb_frame
[robot_state_publisher-3] [INFO] [1605534787.285617054] [robot_state_publisher]: got segment camera_rgb_optical_frame
[robot_state_publisher-3] [INFO] [1605534787.285622492] [robot_state_publisher]: got segment caster_back_left_link
[robot_state_publisher-3] [INFO] [1605534787.285627671] [robot_state_publisher]: got segment caster_back_right_link
[robot_state_publisher-3] [INFO] [1605534787.285633145] [robot_state_publisher]: got segment imu_link
[robot_state_publisher-3] [INFO] [1605534787.285638606] [robot_state_publisher]: got segment wheel_left_link
[robot_state_publisher-3] [INFO] [1605534787.285644188] [robot_state_publisher]: got segment wheel_right_link
[map_server-5] [INFO] [1605534787.313807464] [map_server]: 
[map_server-5] 	map_server lifecycle node launched. 
[map_server-5] 	Waiting on external lifecycle transitions to activate
[map_server-5] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[map_server-5] [INFO] [1605534787.313990423] [map_server]: Creating
[map_server-5] [INFO] [1605534787.315172438] [map_server]: Configuring
[amcl-6] [INFO] [1605534787.299140968] [amcl]: 
[amcl-6] 	amcl lifecycle node launched. 
[amcl-6] 	Waiting on external lifecycle transitions to activate
[amcl-6] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[amcl-6] [INFO] [1605534787.299284325] [amcl]: Creating
[amcl-6] [INFO] [1605534787.358173458] [amcl]: Configuring
[amcl-6] [INFO] [1605534787.358265319] [amcl]: initTransforms
[lifecycle_manager-7] [INFO] [1605534787.287577790] [lifecycle_manager_localization]: Creating
[lifecycle_manager-7] [INFO] [1605534787.294258412] [lifecycle_manager_localization]: Creating and initializing lifecycle service clients
[lifecycle_manager-7] [INFO] [1605534787.296187584] [lifecycle_manager_localization]: Starting managed nodes bringup...
[lifecycle_manager-7] [INFO] [1605534787.296247086] [lifecycle_manager_localization]: Configuring map_server
[lifecycle_manager-7] [INFO] [1605534787.357826674] [lifecycle_manager_localization]: Configuring amcl
[controller_server-8] [INFO] [1605534787.308520319] [controller_server]: 
[controller_server-8] 	controller_server lifecycle node launched. 
[controller_server-8] 	Waiting on external lifecycle transitions to activate
[controller_server-8] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[controller_server-8] [INFO] [1605534787.310185597] [controller_server]: Creating controller server
[controller_server-8] [INFO] [1605534787.358525568] [local_costmap.local_costmap]: 
[controller_server-8] 	local_costmap lifecycle node launched. 
[controller_server-8] 	Waiting on external lifecycle transitions to activate
[controller_server-8] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[controller_server-8] [INFO] [1605534787.359552160] [local_costmap.local_costmap]: Creating Costmap
[planner_server-9] [INFO] [1605534787.352593313] [planner_server]: 
[planner_server-9] 	planner_server lifecycle node launched. 
[planner_server-9] 	Waiting on external lifecycle transitions to activate
[planner_server-9] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[planner_server-9] [INFO] [1605534787.353410570] [planner_server]: Creating
[recoveries_server-10] [INFO] [1605534787.346533670] [recoveries_server]: 
[recoveries_server-10] 	recoveries_server lifecycle node launched. 
[recoveries_server-10] 	Waiting on external lifecycle transitions to activate
[recoveries_server-10] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[amcl-6] [INFO] [1605534787.366999474] [amcl]: initPubSub
[amcl-6] [INFO] [1605534787.372232951] [amcl]: Subscribed to map topic.
[lifecycle_manager-7] [INFO] [1605534787.378308151] [lifecycle_manager_localization]: Activating map_server
[lifecycle_manager-7] [INFO] [1605534787.379508634] [lifecycle_manager_localization]: Activating amcl
[lifecycle_manager-7] [INFO] [1605534787.380258635] [lifecycle_manager_localization]: Managed nodes are active
[map_server-5] [INFO] [1605534787.378475264] [map_server]: Activating
[amcl-6] [INFO] [1605534787.379812741] [amcl]: Activating
[amcl-6] [WARN] [1605534787.379853786] [amcl]: Publishing the particle cloud as geometry_msgs/PoseArray msg is deprecated, will be published as nav2_msgs/ParticleCloud in the future
[amcl-6] [INFO] [1605534787.397813028] [amcl]: Received a 384 X 384 map @ 0.050 m/pix
[bt_navigator-11] [INFO] [1605534787.424749745] [bt_navigator]: 
[bt_navigator-11] 	bt_navigator lifecycle node launched. 
[bt_navigator-11] 	Waiting on external lifecycle transitions to activate
[bt_navigator-11] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[bt_navigator-11] [INFO] [1605534787.427062688] [bt_navigator]: Creating
[planner_server-9] [INFO] [1605534787.439494163] [global_costmap.global_costmap]: 
[planner_server-9] 	global_costmap lifecycle node launched. 
[planner_server-9] 	Waiting on external lifecycle transitions to activate
[planner_server-9] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[planner_server-9] [INFO] [1605534787.439915075] [global_costmap.global_costmap]: Creating Costmap
[lifecycle_manager-13] [INFO] [1605534787.448629538] [lifecycle_manager_navigation]: Creating
[lifecycle_manager-13] [INFO] [1605534787.468485637] [lifecycle_manager_navigation]: Creating and initializing lifecycle service clients
[waypoint_follower-12] [INFO] [1605534787.469619800] [waypoint_follower]: 
[waypoint_follower-12] 	waypoint_follower lifecycle node launched. 
[waypoint_follower-12] 	Waiting on external lifecycle transitions to activate
[waypoint_follower-12] 	See https://design.ros2.org/articles/node_lifecycle.html for more information.
[waypoint_follower-12] [INFO] [1605534787.470113527] [waypoint_follower]: Creating
[lifecycle_manager-13] [INFO] [1605534787.474445519] [lifecycle_manager_navigation]: Starting managed nodes bringup...
[lifecycle_manager-13] [INFO] [1605534787.474489801] [lifecycle_manager_navigation]: Configuring controller_server
[controller_server-8] [INFO] [1605534787.581258824] [controller_server]: Configuring controller interface
[controller_server-8] [INFO] [1605534787.581587375] [controller_server]: Controller frequency set to 20.0000Hz
[controller_server-8] [INFO] [1605534787.581604946] [local_costmap.local_costmap]: Configuring
[controller_server-8] [INFO] [1605534787.584740341] [local_costmap.local_costmap]: Using plugin "voxel_layer"
[controller_server-8] [INFO] [1605534787.588506377] [local_costmap.local_costmap]: Subscribed to Topics: scan
[controller_server-8] [INFO] [1605534787.594965997] [local_costmap.local_costmap]: Initialized plugin "voxel_layer"
[controller_server-8] [INFO] [1605534787.595008777] [local_costmap.local_costmap]: Using plugin "inflation_layer"
[controller_server-8] [INFO] [1605534787.595275723] [local_costmap.local_costmap]: Initialized plugin "inflation_layer"
[controller_server-8] [INFO] [1605534787.606990782] [controller_server]: Created progress_checker : progress_checker of type nav2_controller::SimpleProgressChecker
[controller_server-8] [INFO] [1605534787.607803061] [controller_server]: Created goal_checker : goal_checker of type nav2_controller::SimpleGoalChecker
[controller_server-8] [INFO] [1605534787.610216938] [controller_server]: Created controller : FollowPath of type dwb_core::DWBLocalPlanner
[controller_server-8] [INFO] [1605534787.610648240] [controller_server]: Setting transform_tolerance to 0.200000
[controller_server-8] [INFO] [1605534787.644898878] [controller_server]: Using critic "RotateToGoal" (dwb_critics::RotateToGoalCritic)
[controller_server-8] [INFO] [1605534787.645277237] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.645384295] [controller_server]: Using critic "Oscillation" (dwb_critics::OscillationCritic)
[controller_server-8] [INFO] [1605534787.645568040] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.645621887] [controller_server]: Using critic "BaseObstacle" (dwb_critics::BaseObstacleCritic)
[controller_server-8] [INFO] [1605534787.645667808] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.645716000] [controller_server]: Using critic "GoalAlign" (dwb_critics::GoalAlignCritic)
[controller_server-8] [INFO] [1605534787.645853045] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.645920695] [controller_server]: Using critic "PathAlign" (dwb_critics::PathAlignCritic)
[controller_server-8] [INFO] [1605534787.646030570] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.646077621] [controller_server]: Using critic "PathDist" (dwb_critics::PathDistCritic)
[controller_server-8] [INFO] [1605534787.646169560] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.646217174] [controller_server]: Using critic "GoalDist" (dwb_critics::GoalDistCritic)
[controller_server-8] [INFO] [1605534787.646300952] [controller_server]: Critic plugin initialized
[controller_server-8] [INFO] [1605534787.646317510] [controller_server]: Controller Server has FollowPath  controllers available.
[lifecycle_manager-13] [INFO] [1605534787.658597748] [lifecycle_manager_navigation]: Configuring planner_server
[planner_server-9] [INFO] [1605534787.659123994] [planner_server]: Configuring
[planner_server-9] [INFO] [1605534787.659177442] [global_costmap.global_costmap]: Configuring
[planner_server-9] [INFO] [1605534787.664392186] [global_costmap.global_costmap]: Using plugin "static_layer"
[planner_server-9] [INFO] [1605534787.669502152] [global_costmap.global_costmap]: Subscribing to the map topic (/map) with transient local durability
[planner_server-9] [INFO] [1605534787.674315452] [global_costmap.global_costmap]: Initialized plugin "static_layer"
[planner_server-9] [INFO] [1605534787.674443161] [global_costmap.global_costmap]: Using plugin "obstacle_layer"
[planner_server-9] [INFO] [1605534787.674742236] [global_costmap.global_costmap]: Subscribed to Topics: scan
[planner_server-9] [INFO] [1605534787.685593761] [global_costmap.global_costmap]: StaticLayer: Resizing costmap to 384 X 384 at 0.050000 m/pix
[planner_server-9] [INFO] [1605534787.692227541] [global_costmap.global_costmap]: Initialized plugin "obstacle_layer"
[planner_server-9] [INFO] [1605534787.692277269] [global_costmap.global_costmap]: Using plugin "inflation_layer"
[planner_server-9] [INFO] [1605534787.694401631] [global_costmap.global_costmap]: Initialized plugin "inflation_layer"
[planner_server-9] [INFO] [1605534787.706014683] [planner_server]: Created global planner plugin GridBased of type nav2_navfn_planner/NavfnPlanner
[planner_server-9] [INFO] [1605534787.706055903] [planner_server]: Configuring plugin GridBased of type NavfnPlanner
[planner_server-9] [INFO] [1605534787.707116028] [planner_server]: Planner Server has GridBased  planners available.
[lifecycle_manager-13] [INFO] [1605534787.714582874] [lifecycle_manager_navigation]: Configuring recoveries_server
[recoveries_server-10] [INFO] [1605534787.714847265] [recoveries_server]: Configuring
[recoveries_server-10] [INFO] [1605534787.719186876] [recoveries_server]: Creating recovery plugin spin of type nav2_recoveries/Spin
[recoveries_server-10] [INFO] [1605534787.720535315] [recoveries_server]: Configuring spin
[recoveries_server-10] [INFO] [1605534787.725821361] [recoveries_server]: Creating recovery plugin backup of type nav2_recoveries/BackUp
[recoveries_server-10] [INFO] [1605534787.727269503] [recoveries_server]: Configuring backup
[recoveries_server-10] [INFO] [1605534787.733331499] [recoveries_server]: Creating recovery plugin wait of type nav2_recoveries/Wait
[recoveries_server-10] [INFO] [1605534787.734334197] [recoveries_server]: Configuring wait
[lifecycle_manager-13] [INFO] [1605534787.740406933] [lifecycle_manager_navigation]: Configuring bt_navigator
[bt_navigator-11] [INFO] [1605534787.740685546] [bt_navigator]: Configuring
[bt_navigator-11] [INFO] [1605534787.779174154] [bt_navigator_rclcpp_node]: Waiting for "compute_path_to_pose" action server
[bt_navigator-11] [INFO] [1605534787.779376285] [bt_navigator_rclcpp_node]: "ComputePathToPose" BtActionNode initialized
[bt_navigator-11] [INFO] [1605534787.780662359] [bt_navigator_rclcpp_node]: Waiting for "global_costmap/clear_entirely_global_costmap" service
[bt_navigator-11] [INFO] [1605534787.780801872] [bt_navigator_rclcpp_node]: "ClearGlobalCostmap-Context" BtServiceNode initialized
[bt_navigator-11] [INFO] [1605534787.785593601] [bt_navigator_rclcpp_node]: Waiting for "follow_path" action server
[bt_navigator-11] [INFO] [1605534787.785722491] [bt_navigator_rclcpp_node]: "FollowPath" BtActionNode initialized
[bt_navigator-11] [INFO] [1605534787.786739963] [bt_navigator_rclcpp_node]: Waiting for "local_costmap/clear_entirely_local_costmap" service
[bt_navigator-11] [INFO] [1605534787.786896059] [bt_navigator_rclcpp_node]: "ClearLocalCostmap-Context" BtServiceNode initialized
[bt_navigator-11] [INFO] [1605534787.788435086] [bt_navigator_rclcpp_node]: Waiting for "local_costmap/clear_entirely_local_costmap" service
[bt_navigator-11] [INFO] [1605534787.788558511] [bt_navigator_rclcpp_node]: "ClearLocalCostmap-Subtree" BtServiceNode initialized
[bt_navigator-11] [INFO] [1605534787.789701222] [bt_navigator_rclcpp_node]: Waiting for "global_costmap/clear_entirely_global_costmap" service
[bt_navigator-11] [INFO] [1605534787.789793566] [bt_navigator_rclcpp_node]: "ClearGlobalCostmap-Subtree" BtServiceNode initialized
[bt_navigator-11] [INFO] [1605534787.794137012] [bt_navigator_rclcpp_node]: Waiting for "spin" action server
[bt_navigator-11] [INFO] [1605534787.794300163] [bt_navigator_rclcpp_node]: "Spin" BtActionNode initialized
[bt_navigator-11] [INFO] [1605534787.798397109] [bt_navigator_rclcpp_node]: Waiting for "wait" action server
[bt_navigator-11] [INFO] [1605534787.799248332] [bt_navigator_rclcpp_node]: "Wait" BtActionNode initialized
[lifecycle_manager-13] [INFO] [1605534787.799708061] [lifecycle_manager_navigation]: Configuring waypoint_follower
[waypoint_follower-12] [INFO] [1605534787.799909267] [waypoint_follower]: Configuring
[lifecycle_manager-13] [INFO] [1605534787.814436284] [lifecycle_manager_navigation]: Activating controller_server
[controller_server-8] [INFO] [1605534787.814579662] [controller_server]: Activating
[controller_server-8] [INFO] [1605534787.814603825] [local_costmap.local_costmap]: Activating
[controller_server-8] [INFO] [1605534787.814613553] [local_costmap.local_costmap]: Checking transform
[controller_server-8] [INFO] [1605534787.814640479] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
[controller_server-8] [INFO] [1605534788.314705874] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
[rviz2-4] [INFO] [1605534788.337316691] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-4] [INFO] [1605534788.337509985] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2-4] [INFO] [1605534788.357361925] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-4] [WARN] [1605534788.498757339] [rcl.logging_rosout]: Publisher already registered for provided node name. If this is due to multiple nodes with the same name then all logs for that logger name will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[rviz2-4] [WARN] [1605534788.510852313] [rcl.logging_rosout]: Publisher already registered for provided node name. If this is due to multiple nodes with the same name then all logs for that logger name will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[rviz2-4] [WARN] [1605534788.529816708] [rcl.logging_rosout]: Publisher already registered for provided node name. If this is due to multiple nodes with the same name then all logs for that logger name will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[rviz2-4] [INFO] [1605534788.570599841] [rviz2]: Waiting for the lifecycle_manager_navigation/is_active service...
[rviz2-4] [INFO] [1605534788.570797473] [rviz2]: Sending lifecycle_manager_navigation/is_active request
[rviz2-4] [INFO] [1605534788.571369953] [rviz2]: Trying to create a map of size 384 x 384 using 1 swatches
[rviz2-4] [ERROR] [1605534788.587226059] [rviz2]: Vertex Program:rviz/glsl120/indexed_8bit_image.vert Fragment Program:rviz/glsl120/indexed_8bit_image.frag GLSL link result : 
[rviz2-4] active samplers with a different type refer to the same texture image unit
[controller_server-8] [INFO] [1605534788.814694183] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
[controller_server-8] [INFO] [1605534789.314725818] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
[rviz2-4] [INFO] [1605534789.571181821] [rviz2]: Waiting for the lifecycle_manager_navigation/is_active service...
[rviz2-4] [INFO] [1605534789.571462711] [rviz2]: Sending lifecycle_manager_navigation/is_active request
[controller_server-8] [INFO] [1605534789.814769573] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
[controller_server-8] [INFO] [1605534790.314779043] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
[rviz2-4] [INFO] [1605534790.571815358] [rviz2]: Waiting for the lifecycle_manager_navigation/is_active service...
[rviz2-4] [INFO] [1605534790.572147448] [rviz2]: Sending lifecycle_manager_navigation/is_active request
```

**Comment by SteveMacenski** [comment]
Have you modified anything off default? Are you sure gazebo is running (e.g. do you see the gazebo window is fully loaded and updating laser scans)? That's an error that it can't get odometry, so gazebo's plugins for odometry might not be running / up or crashed. Did you also set the initial pose as the instructions list? The other big reason that TF complains on startup is if you didn't set an initial pose, which is required.

**Comment by mag-seasony** [comment]
I did not modify anything from the default, gazebo manage to run after few mins, but the sdf is not loaded. I did not set the initial pose (my fault as I did not see it in the tutorial). 
Thanks for the help, It is working right now.
The 2 things I had to do for making it work is to indeed set the initial pos (note: isn't it possible to set it by default?), and to download the models from the github as the foxy version does not seem to be available through apt


Again, thanks for the help,
cheers!
